.claude/rules/TDD_COMMIT_WORKFLOW.md

```
@@ -0,0 +1,35 @@
+# TDD & Step-by-Step Commit Workflow Rules
+
+## 目的 (Objective)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)